iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveTableUtil.java

```diff
@@ -225,4 +233,30 @@ public static boolean isCtas(Properties properties) {
     return Boolean.parseBoolean(properties.getProperty(hive_metastoreConstants.TABLE_IS_CTAS));
   }
 
+  static Properties getSerializationProps() {
+    Properties props = new Properties();
+    props.put(serdeConstants.SERIALIZATION_FORMAT, "" + Utilities.tabCode);
+    props.put(serdeConstants.SERIALIZATION_NULL_FORMAT, "NULL");
+    return props;
+  }
+
+  static String getParseData(String parseData, String specId, ObjectMapper mapper, Integer currentSpecId)
+      throws JsonProcessingException {
+    Map<String, String> map = mapper.readValue(parseData, Map.class);
+    String partString =
+        map.entrySet().stream().filter(entry -> entry.getValue() != null).map(java.lang.Object::toString)
```

Comment:
formatting, put filter and map on next separate lines

Reply:
done